ci/unsupported-devices.txt

```diff
@@ -1,6 +1,7 @@
 # Detection models
 person-detection-0106,GPU
 # Segmentation models
+instance-segmentation-person-0007,GPU
```

Comment:
@DmitriySidnev do not use ci/unsupported-devices.txt (we have to delete this), use device_support.md table instead (this located at OMZ/models/public and OMZ/models/intel folders respectively

Reply:
Changes are reverted.